src/components/landing-page/courses-section/index.jsx

```
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)